icu4j/main/charset/src/test/java/com/ibm/icu/dev/test/charset/TestCharset.java

```diff
@@ -1622,9 +1622,9 @@ public void TestCanConvert(/*String encoding*/)throws Exception {
 
     @Test
     public void TestAvailableCharsets() {
-        SortedMap map = Charset.availableCharsets();
-        Set keySet = map.keySet();
-        Iterator iter = keySet.iterator();
+        SortedMap<String,Charset> map = Charset.availableCharsets();
```

Comment:
nit: add space after comma

Reply:
Done (which, of course, nullifies the approval :-)